WordPress/src/main/res/values/strings.xml

```diff
@@ -1345,6 +1345,11 @@
     <string name="reader_discover_empty_button_text">Get Started</string>
     <string name="reader_discover_no_posts_button_text">Follow topics</string>
     <string name="reader_author_with_blog_name" translatable="false">%1$s ▸ %2$s</string>
+    <string name="reader_follow_comments_null_response">Null response received</string>
+    <string name="reader_follow_comments_bad_format_response">Bad format response received</string>
```

Comment:
And similarly here, maybe "Invalid response received" will sound a bit more friendly (if it's even possible with error messages :D)

Reply:
Changed in 498ddfad2d52c1f1bc7d3acf8a1974d26afe3e47